10/umbraco-deploy/deployment-workflow/deploying-changes.md

```diff
@@ -43,10 +43,10 @@ git pull
 git push
 ```
 
-If you had to pull down new commits, it is a good idea to see if any of these commits contained changes to the schema (anything in `umbraco/Deploy/Revision/`). In order to ensure that your local site is up-to-date, and your changes work with the updated schema, you can navigate to the `umbraco/Deploy/` folder and create a deploy marker if one doesn't already exist. From a command line type the following command:
+When pulling new commits, it is a good idea to see if any of these commits contained changes to the schema (anything in `umbraco/Deploy/Revision/`). To ensure your local site is up-to-date, confirm compatibility with the updated schema. You can do this by navigating to the `umbraco/Deploy/` folder and creating a deploy marker if it doesn't exist. From a command line type the following command:
```

Comment:
We run this operation to see is schema is updated locally, not to verify the compatibility, so I made some suggestions here:
```suggestion
When pulling new commits, it is a good idea to see if any of these commits contained changes to the schema (anything in `umbraco/Deploy/Revision/`). To ensure your local schema is up-to-date, you can navigate to the `umbraco/Deploy/` folder and create a deploy marker if it doesn't exist. From a command line type the following command:
```